teachertool/src/components/EvalResultDisplay.tsx

```diff
@@ -27,13 +27,13 @@ const ResultsHeader: React.FC = () => {
     );
 };
 
-export const EvalResultDisplay: React.FC<{}> = () => {
+export const EvalResultDisplay: React.FC<{printRef: React.RefObject<HTMLDivElement>}> = ({ printRef }) => {
```

Comment:
nit: For our other components, we've been defining an interface for the props (i.e. `EvalResultDisplayProps`). For consistency, it'd be nice to do that here too. Functionally, I'm not sure if it makes any difference.